### DISCUSSION TITLE
February| Treasury Support Monthly Report

### DISCUSSION THREAD

**Discussion by casinoyoda** [ringdao-community-guild/discussion#112] [General]
I am happy to sync up with everyone on my recent job transition. I have moved from a technical support role to a treasury support role, with Furqan taking over the vacant position of Technical Manager. You could learn the details from these threads: 
- https://github.com/orgs/dcdao/discussions/101
- https://github.com/orgs/dcdao/discussions/106

I will still be involved in some technical support tasks, such as maintaining the message synchronization between Discord and Slack, managing the Profile repository, approving new applicants, and distributing NFTs. Additionally, I will be responsible for fund tracking and distribution, as well as handling funding proposals.

In February, my work mainly involved:
1. Distributing funds to Ambassadors and Advocates: [Transaction](https://app.safe.global/transactions/tx?safe=eth:0x195741BC761B25A89fc7798087652f97f29B306a&id=multisig_0x195741BC761B25A89fc7798087652f97f29B306a_0xd2a5f509495c4de18c5fd458cf776000823bb48b0c593bc0aaaafaffd0f4a9d7)
2. Managing and verifying fund statistics in February.
3. Reviewing DCDAO membership applications and distributing NFTs: [Release #8](https://github.com/dcdao/profiles/pull/57)
4. Daily maintenance of the Discord-Slack message bridge.

**Comment by nblogist** [comment]
Hello @casinoyoda what name are you using on slack? I'd like to schedule a call with you since I'd be taking over the position of technical manager and would like to know the information about the team you managed before I establish communication with the individual team members can you please find a slot here: https://calendly.com/furqanahmed
